client/dashboard/sites/settings/index.tsx

```diff
@@ -38,6 +39,7 @@ export default function SiteSettings( { siteSlug }: { siteSlug: string } ) {
 				<WordPressSettingsSummary site={ site } />
 				<PHPSettingsSummary site={ site } />
 				<AgencySummary site={ site } />
+				<LegacySummary site={ site } settings={ settings } />
```

Comment:
I think this should be `LegacyContactSettingsSummary`, `AgencySettingsSummary` to be consistent with the other. 🤔

Reply:
Agreed. Updated in 90d9dcc10d6654e1d66264173092540aeeaab1ce.